tests/autoyast/installation.pm

```diff
@@ -138,7 +138,7 @@ sub run {
     push @needles, 'autoyast-error' unless get_var('AUTOYAST_EXPECT_ERRORS');
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
     # So push a needle to check upcoming reboot on zVM that is a way to indicate the stage done
-    push @needles, 'autoyast-stage1-reboot-upcoming' if check_var('ARCH', 's390x') || is_pvm;
+    push @needles, 'autoyast-stage1-reboot-upcoming' if check_var('ARCH', 's390x') || (is_pvm && !is_upgrade);
```

Comment:
Hmm, I'm not sure why we need this change, as this reboot is required to handle second stage. From you VR it looks like there is no second stage, but in any case this should not be backend specific. For such cases, I would suggest to handle it later and exit before second stage loop, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11844/files#diff-269577be56cf62326b2be1ff7b71693c5b5aafd0cf37a5ebf7492d63bb9a0245R332

Reply:
After looking at your code, seems that we can improve it for the autoyast installations on powerVM, as in our case we don't have this pop-up (https://openqa.suse.de/tests/5341502#step/installation/9), so we cannot rely on it: https://openqa.suse.de/tests/5337658
Do you know what causes this behavior? In any case, as stated above, I believe that it is same behavior on s390x too, so is not powerVM specific.

Reply:
If I don't change this part. It cannot restart system: https://openqa.suse.de/tests/5341423#step/installation/10. It seems it didn't finish to install system then it reboot system.  I don't know why it will pop reboot screen. So I just change it for upgrade cases.

Reply:
I guess it's due to `reboot_timeout=0` boot option, which we can use in all scenarios. As of now this change is safe, but in general handling here likely depends on that boot option and not just backend. Same can be applied for s390x.

Reply:
I think it is really different here for s390x and power, since it hasn't finished installation for power so skip here is reasonable. @rwx788 WDYT?

Reply:
As per comment from @tinawang123 in this PR, seems that this behavior depends on the settings in the profile, which is more realistic. I will check myself if we can apply this solution for complex cases like s390x and powerVM, as those platforms have a challenge to detect when reboot will happen in order to successfully reconnect to them.
As stated before, this change is safe and won't break other tests, even though we have confirmed that it's not about powerVM here, but rather about the profile. So now you will need to assure that profile contains `security` section defined as proposed here.